com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -70,6 +70,29 @@ private enum SMAAStage
         readonly GraphicsFormat m_LutFormat;
         readonly HableCurve m_HableCurve;
 
+        //Viewport information
+
+        Vector2Int m_FullRes = new Vector2Int(1,1);
+        Vector2Int m_DownsampledRes = new Vector2Int(1,1);
+
+        internal enum ResolutionGroup
+        {
+            Downsampled,
+            Full
+        }
+
+        internal ResolutionGroup ResGroup { set; get; }
```

Comment:
This should be camelCase as per conventions (same for below Viewport and UpsamplerSchedule)

Reply:
I wonder though, can these be private instead?

Reply:
done, privatizing :D